supervisor/api/__init__.py

```diff
@@ -228,6 +228,12 @@ def _register_resolution(self) -> None:
         self.webapp.add_routes(
             [
                 web.get("/resolution/info", api_resolution.info),
+                web.post(
+                    "/resolution/check/{check}/disable", api_resolution.disable_check
```

Comment:
Do we expect other options to become available and so would a generic `/options` endpoint be better? How do we do it for other APIs in the Supervisor (addons etc)

Reply:
Checks are "stupid", they run or they don't, if we want options that are not pr. check but on resolution or resolution/check